src/Util/ModuleUtils.php

```diff
@@ -65,6 +70,30 @@ public function getVersions(string $moduleName, bool $withAssetOnly = true): arr
         ), $versions);
     }
 
+    public function getFromBucket(): ModuleCollection
+    {
+        $modules = $this->bucket->objects(['prefix' => 'assets/modules/']);
+        $list = [];
+        foreach ($modules as $module) {
+            $parts = explode('/', $module->info()['name']);
+            if (!isset($list[$parts[count($parts) - 3]])) {
+                $list[$parts[count($parts) - 3]] = [];
+            }
+            $list[$parts[count($parts) - 3]][] = $parts[count($parts) - 2];
+        }
+
+        $tet = [];
```

Comment:
Tet ? 😄

Reply:
That's a temporary name that successfully persisted 😆